live-build/misc/upgrade-scripts/verify

```diff
@@ -41,6 +41,20 @@ function cleanup() {
 	#
 	[[ -z "$CONTAINER" ]] && return
 
+	#
+	# We need to collect logs within the container and move to
+	# container's host otherwise those will delete when container
+	# is deleted. We leverage support_info script to do the collection
+	# and only collect "log" and "dropbox" types. User who wish to add
+	# more collection types from support_info here should check if
+	# they work in the container.
+	# We collect only one copy to limit space usage.
+	#
+	report_progress 70 "Collecting verification logs"
+	rm -rf "${UPDATE_VERIFY_LOG_DIR:?}/*"
+	mkdir -p "${UPDATE_VERIFY_LOG_DIR}"
```

Comment:
Instead of using a single "global" directory to store logs, how about we use a per-container directory? This way, we could potentially have multiple processes running upgrade verification at the same time, and they won't clobber each other when writing their logs. e.g. use a directory like this:
```
mkdir -p "${UPDATE_VERIFY_LOG_DIR}/${CONTAINER}"
${SUPPORT_INFO_SCRIPT} -o "${UPDATE_VERIFY_LOG_DIR}/${CONTAINER}" ...
```

While we might not run multiple upgrade verify's concurrently right now, since all the other code would allow it, it'd be a shame to break that just because of this log collection.

Also, if we do make the log collection use container specific directories, it means (theoretically) we could have more logs collected at any given time, which means that we might need more disk space to store them. Further, it means that we'd have to clean up the per-container directories at some point (e.g. the old logs that aren't needed anymore).

If this per-container log directory makes things more complicated for little benefit, then at least we could do something like this:
```
mkdir -p "${UPDATE_VERIFY_LOG_DIR}/${CONTAINER}"
${SUPPORT_INFO_SCRIPT} -o "${UPDATE_VERIFY_LOG_DIR}/${CONTAINER}" -t "log dropbox"
mv "${UPDATE_VERIFY_LOG_DIR}/${CONTAINER}" "${UPDATE_VERIFY_LOG_DIR}/latest"
```
This way, if multiple verify's were running concurrently, we'd at least atomically get the logs for one of the runs or the other.. rather than now, where we could get some logs from one run, and some logs from the other run.

What do you think?